src/main/kotlin/xyz/bluspring/kilt/loader/KiltLoader.kt

```diff
@@ -381,7 +383,7 @@ class KiltLoader {
                 if (FabricLoader.getInstance()
                         .isModLoaded(mod.modId) || FabricLoaderImpl.INSTANCE.getModCandidate(mod.modId) != null
                 ) {
-                    Kilt.logger.warn("Duplicate Forge and Fabric mod IDs detected: ${mod.modId}")
+                    Kilt.logger.warn("Duplicate Fabric and Forge mod IDs detected: ${mod.modId}")
```

Comment:
I don't see any point in switching these around, honestly.

Reply:
That was just an IDE warning for duplicate log messages, albeit not that useful since this log probably won't get reached if the earlier (duplicate) one is called (i think?)

Reply:
Unfortunately this one is required, as sometimes it manages to miss that first one, hence why the second one is included.